pkg/rthooks/rthooks.go

```diff
@@ -10,19 +10,18 @@
 // RuntimeHookRequest is issued.
 //
 // Specifically:
-//  - sensors can register their callbacks at init() using RegisterCallbacksAtInit
```

Comment:
I think this is `gofmt` doing this in my editor.

Reply:
Yap, it seems to be `gofmt`:

```diff
diff pkg/rthooks/rthooks.go.orig pkg/rthooks/rthooks.go                                                                                                                                                                                                                                                                        
--- pkg/rthooks/rthooks.go.orig                                                                                                                                                                                                                                                                                                
+++ pkg/rthooks/rthooks.go                                                                                                                                                                                                                                                                                                     
@@ -10,18 +10,18 @@                                                                                                                                                                                                                                                                                                            
 // RuntimeHookRequest is issued.                                                                                                                                                                                                                                                                                              
 //                                                                                                                                                                                                                                                                                                                            
 // Specifically:                                                                                                                                                                                                                                                                                                              
-//  - sensors can register their callbacks at init() using RegisterCallbacksAtInit                                                                                                                                                                                                                                            
-//    which registers in these hooks in globalRunner.                                                                                                                                                                                                                                                                         
-//  - after init(), GlobalRunner() can be used to retrieve this runner and pass                                                                                                                                                                                                                                               
-//    it to the gRPC server code so that it can execute these callbacks when a                                                                                                                                                                                                                                                
-//    RuntimeHookRequest is issued.                                                                                                                                                                                                                                                                                           
-//  - some of these hooks need access to pkg/watcher, so before passing the                                                                                                                                                                                                                                                   
-//    runner to gRPC server, we add the watcher as well. Hooks can access the                                                                                                                                                                                                                                                 
-//    watcher via the argument passed in the executed callback.                                                                                                                                                                                                                                                               
-//  - all callbacks are executed, i.e., if a callback returns an error execution of callbacks does                                                                                                                                                                                                                            
-//    not stop                                                                                                                                                                                                                                                                                                                
-//  - if any callback fails with an error, the gRPC server will return an error to the client (see                                                                                                                                                                                                                            
-//    pkg/server/server.go)                                                                                                                                                                                                                                                                                                   
+//   - sensors can register their callbacks at init() using RegisterCallbacksAtInit                                                                                                                                                                                                                                           
+//     which registers in these hooks in globalRunner.                                                                                                                                                                                                                                                                        
+//   - after init(), GlobalRunner() can be used to retrieve this runner and pass                                                                                                                                                                                                                                              
+//     it to the gRPC server code so that it can execute these callbacks when a                                                                                                                                                                                                                                               
+//     RuntimeHookRequest is issued.                                                                                                                                                                                                                                                                                          
+//   - some of these hooks need access to pkg/watcher, so before passing the                                                                                                                                                                                                                                                  
+//     runner to gRPC server, we add the watcher as well. Hooks can access the                                                                                                                                                                                                                                                
+//     watcher via the argument passed in the executed callback.                                                                                                                                                                                                                                                              
+//   - all callbacks are executed, i.e., if a callback returns an error execution of callbacks does                                                                                                                                                                                                                           
+//     not stop                                                                                                                                                                                                                                                                                                               
+//   - if any callback fails with an error, the gRPC server will return an error to the client (see                                                                                                                                                                                                                           
+//     pkg/server/server.go)                                                                                                                                                                                                                                                                                                  
 package rthooks                                                                                                                                                                                                                                                                                                               
                                                                                                                                                                                                                                                                                                                               
 import (       
 ```

Reply:
Maybe a bit nitpicky but could we pick this out and squash it into the first commit to avoid the noise here?

Reply:
The reason, I have it in a different patch is that the:
> +//  - if any callback fails with an error, the gRPC server will return an error to the client (see                                                            
> +//    pkg/server/server.go)

is not true before. It becomes true after the changes are applied in the second patch where we actually return an error from the server.